### ISSUE TITLE
Incorrect Chapter reference in LinkedIn Notebook

### ISSUE THREAD

**Issue by edglazer** [mikhailklassen/Mining-the-Social-Web-3rd-Edition#5]
Awesome book and Jupyter notebooks!
There are a couple spots where the notebook for Chapter 4 on LinkedIn references the subdirectory as ch03-linked - though it is actually ch04-linkedin. If I knew how I'd create a pull request, sorry not to be more helpful!

**Comment by karelin** [comment]
Similar thing in Chapter 3 on Instagram, and in Chapter 5 with Video course on Google+ (if it is still necessary). And notebook for Appendix B refers to non-existing resources for Chapter 2.

**Comment by ptwobrussell** [comment]
@edglazer @karelin - Thanks for the help in tracking down these references that got overlooked. I really appreciate it!

@mikhailklassen - Should we remove the Google+ (video course) notebook entirely now that the deprecation date for Google+ is so close? That's the only thing left to consider in closing this issue as far as I know.

**Comment by mikhailklassen** [comment]
@edglazer - I fixed the references to ch03 resources and pointed them to ch04. Thanks for catching that. Some of the chapters got resequenced and we missed those references.

@karelin @ptwobrussell - I've now removed the Google+ (Video Course) notebook, since the lesson material for that notebook is close to that of Mining Text Files. Also, I had forgotten that users who purchased access to the video course on O'Reilly get those notebooks as part of the course files, so this repo can focus entirely on the book-related content.
